scripts/update-holonix-version.sh

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
```

Comment:
```suggestion
set -euox pipefail
```

would be good at least initially